PocketKit/Sources/PocketKit/MyList/ItemsList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -615,6 +615,14 @@ extension SavedItemsListViewModel {
 
         switch filter {
         case .listen:
+            // If the user selected a filter, and the user is not in the Listen tags playlist feature flag
+            // Remove any filters and sorts they selected and re-fetch data before showing listen.
```

Comment:
My interpretation: if a user has a tag / other filter selected, if they tap the listen button and they are _not_ in the playlist group, this will "reset" the list? This is just so the "playlist" is the entire list, correct? If my interpretation is correctly, I feel like this _might_ be jarring as a user to see their view "reset", but I think I understand its purpose.

Reply:
@dskuza correct. This is what we want to do on purpose for now. We will revert this behavior in a point release. Can explain the reasoning in slack.